src/CLRunner.ts

```diff
@@ -290,7 +290,7 @@ export class CLRunner {
         }
     }
 
-    public async StartSessionAsync(state: CLState, conversationRef: Partial<BB.ConversationReference> | null, appId: string, sessionStartFlags: SessionStartFlags, createParams: CLM.SessionCreateParams | CLM.CreateTeachParams): Promise<CLM.Teach | CLM.Session> {
+    public async CreateSessionAsync(state: CLState, conversationRef: Partial<BB.ConversationReference> | null, appId: string, sessionStartFlags: SessionStartFlags, createParams: CLM.SessionCreateParams | CLM.CreateTeachParams): Promise<CLM.Teach | CLM.Session> {
```

Comment:
>CreateSessionAsync [](start = 17, length = 18)

might be better to move CreateSession, StartSession, SessionExtract, SessionScore function to a session manager object that holds the ILogStoage.